filament/backend/src/vulkan/VulkanPipelineCache.cpp

```diff
@@ -423,31 +464,48 @@ void VulkanPipelineCache::bindProgramBundle(const ProgramBundle& bundle) noexcep
 }
 
 void VulkanPipelineCache::bindRasterState(const RasterState& rasterState) noexcept {
-    VkPipelineRasterizationStateCreateInfo& raster0 = mPipelineKey.rasterState.rasterization;
-    const VkPipelineRasterizationStateCreateInfo& raster1 = rasterState.rasterization;
-    VkPipelineColorBlendAttachmentState& blend0 = mPipelineKey.rasterState.blending;
-    const VkPipelineColorBlendAttachmentState& blend1 = rasterState.blending;
-    VkPipelineDepthStencilStateCreateInfo& ds0 = mPipelineKey.rasterState.depthStencil;
-    const VkPipelineDepthStencilStateCreateInfo& ds1 = rasterState.depthStencil;
-    VkPipelineMultisampleStateCreateInfo& ms0 = mPipelineKey.rasterState.multisampling;
-    const VkPipelineMultisampleStateCreateInfo& ms1 = rasterState.multisampling;
+    auto& raster0 = mPipelineKey.rasterState.rasterization;
```

Comment:
to make the code more readable, you *could* write a standalone `operator=` for each of these struct.

Reply:
Yeah, the sub-structs are anonymous right now but I'll consider doing this in the future.